lib/session.js

```diff
@@ -523,6 +523,12 @@ function session(sock) {
 	// This is intended to be extended
 	returnObj.handleCmd = {};
 
+	// Add a error handler by default
+    returnObj.on('error', function(data) {
+        log.warn('larvitsmpp: lib/session.js: session() - sock.on(error) - Socket error detected!:', data.message || 'No info');
```

Comment:
`data` may be null. `data.message` may throw the undefined error. You should do something like `data? data.message : 'No info'`